front/src/applications/editor/components/EntityError.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import cx from 'classnames';
+import { BsFillExclamationOctagonFill } from 'react-icons/bs';
+import { isNil, uniqueId } from 'lodash';
+
+import { EditorEntity } from 'types';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { getInfraID } from 'reducers/osrdconf/selectors';
+import { InfraErrorLine } from './InfraErrors/InfraError';
+
+const EntityError: FC<{ entity: EditorEntity; className?: string }> = ({ entity, className }) => {
+  const { t } = useTranslation();
+  const infraID = useSelector(getInfraID);
+  const [getEntityError, { data }] = osrdEditoastApi.endpoints.getInfraByIdErrors.useLazyQuery({});
```

Comment:
I don't really know what happens with RTK "conditionnal queries".  
Does the skip parameter is used only on mount or at every query parameter changes ?

Reply:
Every time the parameter changes, it's quite handy, you can use it like this directly (and remove the useEffect)
```
  const { data } = osrdEditoastApi.endpoints.getInfraByIdErrors.useQuery(
    {
      id: infraID as number,
      objectId: entity.properties.id,
    },
    { skip: !infraID }
  );

```

Reply:
1) I try to not use the `as` in my code when it's possible (to let TS handles types). => `id: infraID || -1`
2) I prefer to use `isNil` from lodash to avoid 0, undefined, false JS issue